wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -542,6 +570,9 @@ async def run(
                     LaunchAgent.name()
                 )
 
+            if launch_project.job_base_image:
+                apply_code_mount_configuration(resource_args, launch_project)
```

Comment:
Ah this is CRD path which is why this is uncovered? Can we paramaterize the test to cover this as well?